content/guides/compose-bake/index.md

```diff
@@ -0,0 +1,396 @@
+your Compose file with build configurations that might not be desirable for
+local builds, consider separating the production builds from the local builds
+by using Bake to build images for release. This approach separates concerns:
+using Compose for local development and Bake for production-ready builds, while
+still reusing service definitions and fundamental build configurations.
+
+## Build with Bake
+
+Like Compose, Bake parses the build definition for a project from a
+configuration file. Bake supports HashiCorp Configuration Language (HCL), JSON,
+and the Docker Compose YAML format. When you use Bake with multiple files, it
+will find and merge all of the applicable configuration files into one unified
+build configuration. The build options defined in your Compose file are
+extended, or in some cases overridden, by options specified in the Bake file.
+
+The following section explores how you can use Bake to extend the build options
+defined in your Compose file for production.
+
+### View the build configuration
+
+Bake automatically creates a build configuration from the `build` properties of
+your services. Use the `--print` flag for Bake to view the build configuration
+for a given Compose file. This flag evaluates the build configuration and
+outputs the build definition in JSON format.
+
+```console
+$ docker buildx bake --print
+```
+
+The JSON-formatted output shows the group that would be executed, and all the
+targets of that group. A group is a collection of builds, and a target
+represents a single build.
+
+```json
+{
+  "group": {
+    "default": {
+      "targets": ["seed", "vote", "result", "worker"]
+    }
+  },
+  "target": {
+    "result": {
+      "context": "result",
+      "dockerfile": "Dockerfile",
+      "tags": ["username/result"]
+    },
+    "seed": {
+      "context": "seed-data",
+      "dockerfile": "Dockerfile"
+    },
+    "vote": {
+      "context": "vote",
+      "dockerfile": "Dockerfile",
+      "tags": ["username/vote"],
+      "target": "dev"
+    },
+    "worker": {
+      "context": "worker",
+      "dockerfile": "Dockerfile",
+      "tags": ["username/worker"]
+    }
+  }
+}
+```
+
+As you can see, Bake has created a `default` group that includes four targets:
+
+- `seed`
+- `vote`
+- `result`
+- `worker`
+
+This group is created automatically from your Compose file; it includes all of
+your services containing a build configuration. To build this group of services
+with Bake, run:
+
+```console
+$ docker buildx bake
+```
+
+### Customize the build group
+
+Start by redefining the default build group that Bake executes. The current
+default group includes a `seed` target — a Compose service used solely to
+populate the database with mock data. Since this target doesn't produce a
+production image, it doesn’t need to be included in the build group.
+
+To customize the build configuration that Bake uses, create a new file at the
```

Comment:
Should we callout file creation here? Even just a **bold** would help my silly eyes. Just above we did a `bake --print` from the compose YAML and my first instinct here was to pipe the output from that to a new file and modify it.

Reply:
Let me do a `touch docker-bake.hcl` example, just to make it clear that we're creating a new file.